<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>450</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTest">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="result_3">
       <property name="text">
        <string>Goods</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="result_2">
       <property name="text">
        <string>Materials</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table_limits">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>551</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="result_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>149</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Variables</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_max_function">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="get_variables">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Variables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_limit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>260</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_text">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Result :</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_calculation">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="result_variables">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
